R/install.R

```diff
@@ -235,20 +235,30 @@ lantern_install_libs <- function(version, type, install_path, install_config) {
 }
 
 install_type_windows <- function(version) {
+  verbose <- Sys.getenv("TORCH_VERBOSE_INSTALL")
   cuda_version <- NULL
   cuda_path <- Sys.getenv("CUDA_PATH")
 
   if (nzchar(cuda_path)) {
-    versions_file <- file.path(cuda_path, "version.txt")
+    if (verbose == "1") { message(paste0("CUDA_PATH environment variable is set to ", cuda_path)) }
+    versions_file <- file.path(cuda_path, "version.json")
```

Comment:
I think we need test for both files (`version.txt` and  `version.json`) to make sure older versions are also supported.